maestrowf/conductor.py

```diff
@@ -356,7 +355,8 @@ def monitor_study(self):
             dag.write_status(os.path.split(pkl_path)[0])
             # Sleep for SLEEPTIME in args if study not complete.
             if completion_status == StudyStatus.RUNNING:
-                sleep(sleep_time)
+                #sleep(sleep_time)
+                sleep(3)
```

Comment:
You need to change back to the old sleep -- this breaks a feature in Maestro.

Reply:
Oh yeah sorry, I changed that to do some iPython testing. I'll change it back